test/new_relic/http_client_test_cases.rb

```diff
@@ -531,27 +506,42 @@ def test_still_records_tt_node_when_request_fails
     # transaction in which the error occurs. That, coupled with the fact that
     # fixing it for old versions of Typhoeus would require large changes to
     # the instrumentation, makes us say 'meh'.
-    is_typhoeus = (client_name == 'Typhoeus')
-    if !is_typhoeus || (is_typhoeus && Typhoeus::VERSION >= '0.5.4')
-      evil_server = NewRelic::EvilServer.new
-      evil_server.start
+    skip 'Not tested with Typhoeus < 0.5.4' if typhoeus? && Typhoeus::VERSION < '0.5.4'
 
-      in_transaction do
-        begin
-          get_response("http://localhost:#{evil_server.port}")
-        rescue
-          # it's expected that this will raise for some HTTP libraries (e.g.
-          # Net::HTTP). we unfortunately don't know the exact exception class
-          # across all libraries
-        end
+    evil_server = NewRelic::EvilServer.new
+    evil_server.start
+
+    in_transaction do
+      begin
+        get_response("http://localhost:#{evil_server.port}")
+      rescue
+        # it's expected that this will raise for some HTTP libraries (e.g.
+        # Net::HTTP). we unfortunately don't know the exact exception class
+        # across all libraries
       end
+    end
 
-      last_node = find_last_transaction_node()
+    perform_last_node_assertions
 
-      assert_equal("External/localhost/#{client_name}/GET", last_node.metric_name)
+    evil_server.stop
+  end
 
-      evil_server.stop
-    end
+  def typhoeus?
```

Comment:
Do we still need this method?

Reply:
Good call here. Now that we've reverted the stacked Typhoeus/Ethon instrumentation, this helper method is down to just a single caller. Refactored the method out with [285de8a](https://github.com/newrelic/newrelic-ruby-agent/pull/2260/commits/285de8a22b4154a62ecace9ce9856aca7f741161) to go back to the way `dev` currently has it.